CriteoPublisherSdk/Tests/UnitTests/Network/CR_NetworkManagerTests.m

```diff
@@ -138,7 +135,7 @@ - (void)skipped_testNetworkManagerPostCall {
                                nil],
           @"slots", nil];
 
-  NSURL *url = [NSURL URLWithString:@"http://directbidder-test-app.par.preprod.crto.in/"
+  NSURL *url = [NSURL URLWithString:@"https://localhost:9099/directbidder-test-app/"
```

Comment:
This was the URL of CDB preprod. What do you think about putting this in a constant ? So if we need to change it, we can do it easily.

Reply:
It does not makes much sense as it is a one shot use in this test